src/common/copier_config/csv_reader_config.cpp

```diff
@@ -30,6 +30,8 @@ static void bindBoolParsingOption(CSVReaderConfig& config, const std::string& op
         config.option.allowUnbracedList = optionValue;
     } else if (optionName == CopyConstants::IGNORE_ERRORS_OPTION_NAME) {
         config.option.ignoreErrors = optionValue;
+    } else if (optionName == "AUTODETECTION" || optionName == "AUTO_DETECTION") {
```

Comment:
Maybe let's follow duckdb's csv option names?
For other csv option names, we follow their style.
In duckdb, the option name is `auto-detect`.